### PR TITLE
Add employ address issue fixed

### PR DESCRIPTION
**Pull request by AbusiddikAntor**
Manager- add employ -address input field- comma, hyphen ,if you take them,it shows an error

### CODE REVIEW

File: frontend/src/pages/employee/AddEmployee.jsx [hunk 1]
```diff
@@ -29,13 +29,14 @@ import EmployeeView from "./EmployeeView";
...
-      /^[a-zA-Z][a-zA-Z0-9\s]*$/,
-      "Address must start with a character and can include characters and numbers"
-    ),
+  // address: yup
+  //   .string()
+  //   .required("Address is required")
+  //   .matches(
+  //     /^[a-zA-Z][a-zA-Z0-9\s]*$/,
+  //     "Address must start with a character and can include characters and numbers"
+  //   ),
```
Comment (devBasar) [inline frontend/src/pages/employee/AddEmployee.jsx:38]: please remove unused code.

**Review by devBasar — COMMENTED** [review]
@AbusiddikAntor please remove unused code and resubmit. Thanks.

File: frontend/src/pages/employee/AddEmployee.jsx (hunk 1 above)
Comment (AbusiddikAntor) [reply frontend/src/pages/employee/AddEmployee.jsx]: please check my last commit

**Review by devBasar — APPROVED** [review]
unnecessary code is removed.

**Pull request merged**